src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
+  </Collapser>
+</CollapserGroup>
+
+## Fine-tune advanced signal settings [#advanced-signal-settings]
+
+New Relic constantly observes the data that streams from your applications and into our system. But not all applications send signals at the same frequency or cadence. Some events could send signals to our system once a minute while others could only report data to New Relic once every day. An alert condition is a **specific** container designed for a **specific** use case. When creating an alert condition this section is the most customizable for the data that you're evaluating. 
+
+We're going to customize these advanced signal settings for our condition that is looking for web transaction latency issues. 
+
+<CollapserGroup>
+  <Collapser
+    id="window-duration"
+    title="Window duration"
+  >
 
 <img
-  title="The query's visualization shows each host's average CPU utilization."
-  alt="The query's visualization shows each host's average CPU utilization."
-  src={accountsShowNrqlGraph}
+  title="Window duration"
+  alt="A screenshot depicting advanced custom settings for an alert condition."
+  src={alertsWindowDuration}
 />
+  Setting the window duration for your alert condition tells New Relic how to group your data. If you're creating an alert condition for a data set that sends a signal to New Relic once every hour, you'd want to set the window duration to something closer to sixty minutes because it'll help spot patterns and unusual behavior. But, if you're creating an alert condition for web transaction time and New Relic collects a signal for that data every minute, we'd recommend setting the window duration to one minute. 
+  
+  For your first alert we recommend sticking with our default settings but the more you get familiar with creating an alert condition we encourage you to customize these fields based on your own experience. 
+  </Collapser>
 
-### Step 4: Create a policy
+  <Collapser
+    id="sliding-window"
+    title="Use sliding window aggregation"
+  >
 
-Finally, add your condition to an existing policy or create a new one.
+  Throughout the day, data streams from your application into New Relic. Instead of evaluating that data immediately for incidents, alert conditions collect the data over a period of time known as the **aggregation window**. An additional delay allows for slower data points to arrive before the window is aggregated.
 
-<img
-  title="A screenshot of the Policy section for the Create alert form."
-  alt="A screenshot of the Policy section for the Create alert form."
-  src={accountsNrqlCreatePolicy}
+  Sliding windows are helpful when you need to smooth out "spiky" charts. One common use case is to use sliding windows to smooth line graphs that have a lot of variation over short periods of time in cases where the rolling aggregate (for example a rolling mean) is more important than aggregates from narrow windows of time.
```

Comment:
I'm almost certain it is just me but the example in parenthesis throw off the flow of the sentence. I just wanted to bring this up so you knew, not sure what to do or if you even should do anything about it really.

Reply:
I'm just going to delete it. Not worth it!